projects/plugins/jetpack/_inc/lib/class-jetpack-ai-helper.php

```diff
@@ -409,6 +417,7 @@ public static function get_ai_assistance_feature() {
 				'tier-plans-enabled'   => WPCOM\Jetpack_AI\Usage\Helper::ai_tier_plans_enabled(),
 				'costs'                => WPCOM\Jetpack_AI\Usage\Helper::get_costs(),
 				'features-control'     => WPCOM\Jetpack_AI\Feature_Control::get_features(),
+				'chrome-ai-tokens'     => $chrome_ai_tokens,
```

Comment:
Would it make sense for this to be a filter? 

this way we don't have to check WP_CONTENT_DIR . '/lib/jetpack-ai/chrome-ai-tokens.php' is readable . 
but implement but the filter when this is the case.

Reply:
hum, good idea. I'll give it a try.